README.md

```diff
@@ -12,6 +12,7 @@ Inspired and taken (with some liberties) from [Aggregate by Brent](https://githu
 
 | Version | Illuminate  | Status         | PHP Version |
 |:--------|:------------|:---------------|:------------|
+| 4.x     | 10.x        | Active Support | ^8.1        |
```

Comment:
We're actually supporting 9.x and 10.x from the matrix, and php ^8.0 right?

Reply:
Yeah so I figured it would probably be fine to limit support to to the current and previous major Laravel versions.

Laravel 10 requires PHP ^8.1 whereas Laravel 9 supported PHP ^8.0.  TBH I think we should probably just drop that PHP column on the readme anyway.  If someone is on Laravel 10 they'll know they need 8.1 anyway.

The new major version of this package is needed due to the dropped support for L8.

Would you like me to just remove that table column and leave it to the user to sort out their own PHP versions?

Reply:
I would say 8.0|^8.1 or something like that, then the user will figure out that 8.0 is just for L9, what do you say?

Reply:
The main thing for me was to add Illuminate 9.x to the column :)

Reply:
Ah gotcha!  OK have updated the table now - let me know if you need anything else tweaked

Reply:
That's great thanks, gonna release asap 4.0.0.